msbuild/Xamarin.Mac.Tasks/Xamarin.Mac.Common.targets

```diff
@@ -512,7 +512,6 @@ Copyright (C) 2014 Xamarin. All rights reserved.
 			AppBundleName="$(_AppBundleName)"
 			AppBundleDir="$(AppBundleDir)"
 			AppManifest="$(_AppManifest)"
-			Architecture="$(XamMacArch)"
```

Comment:
So this was just dead code, we were not using the architecture at all (:

Reply:
Yea, not used at all